Dokan/Dokan/Scenes/ProductDetails/Cell/ReviewerTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ReviewerTableViewCell.swift
+//  Dokan
+//
+//  Created by raniazeid on 28/07/2022.
+//
+
+import UIKit
+
+class ReviewerTableViewCell: UITableViewCell {
+
+   @IBOutlet weak var reviewerImageView: UIImageView!
+    
+    @IBOutlet weak var reviewerNameLabel: UILabel!
+    
+    @IBOutlet weak var reviewerComment: UITextView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    //MARK: - method to configure cell
+    func configureCell(reviewTableViewCellViewModel:ReviewTableViewCellViewModel){
+        reviewerNameLabel.text = reviewTableViewCellViewModel.reviewerName
+        reviewerComment.text = reviewTableViewCellViewModel.reviewerComment
+        
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    
+}
+struct ReviewTableViewCellViewModel{
+     var reviewerImage : String
+     var reviewerName : String
+     var reviewerComment : String
+ }
```

Comment:
- Let's add it as a nested type inside the cell itself. 
- We prefer immutable properties. 
- 
```suggestion
// MARK: - Nested Types
//
extension ReviewerTableViewCell { 
     struct ViewModel {
          let reviewerImageUrl: String
          let reviewerName: String
          let reviewerComment: String
     }
}
```

Reply:
thanks for this note ,I will edit itI will edit it